96848518 bottles of beer master

andysowar opened the pull request.

andysowar committed:
First working version of bottles_of_beer

[#96848518]

andysowar committed:
Update homework to work recursively & add pluralize method

[#96848518]

andysowar committed:
Fix last two lines of song

[#96848518]

jasonnoble commented on methods_and_recursion/bottles_of_beer.rb:
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+def pluralize(number, string)
+  string += 's' unless number == 1
```
Perfect use of the `unless` method

jasonnoble commented on methods_and_recursion/bottles_of_beer.rb:
```diff
@@ -0,0 +1,18 @@
+
+def pluralize(number, string)
+  string += 's' unless number == 1
+  return string
```
`return` is not needed here, you could just have `string` on a line by itself.

jasonnoble replied:
What if this method returned the number and the string, i.e. "99 bottles" or "No more bottles"

jasonnoble commented on methods_and_recursion/bottles_of_beer.rb:
```diff
@@ -0,0 +1,18 @@
+end
+
+def bottles(beer)
+  puts "No more bottles of beer on the wall, no more bottles of beer.
```
There's a missing `if ... else ... end` here...

andysowar committed:
Made corrections on homework

Removed return form pluralize method & made it display number along with bottles.
Added if/end statement to bottles method.

[#96848518]
